src/char/char.cpp

```diff
@@ -512,6 +512,25 @@ int char_mmo_char_tosql(uint32 char_id, struct mmo_charstatus* p){
 			strcat(save_status, " hotkeys");
 	}
 #endif
+	// item favorites
```

Comment:
With this block of code you're always clearing and writing item favorites. Maybe only do this if there's a change? See friends or hotkeys above.